R/length.R

```diff
@@ -62,12 +68,14 @@
 #'   length = c(10, 8)
 #' )
 #'
-#' adsl <- xportr_length(adsl, metadata, domain = "adsl")
+#' adsl <- xportr_length(adsl, metadata, domain = "adsl", length = "metadata")
 xportr_length <- function(.df,
                           metadata = NULL,
                           domain = NULL,
+                          length = c("metadata", "data"),
```

Comment:
Should we add this parameter after `verbose`?

I understand and support the reason for the current order, however, an error was triggered on our vignettes due to parameter order and I'm a bit concerned about our users facing the same problem

```r
# example
xportr_length(adsl, metadata, "adsl", "message")
#> Error in match.arg(length) : 'arg' should be one of “metadata”, “data”
```

Thoughts?